public/wp-content/themes/glific/front-page.php

```diff
@@ -110,6 +110,95 @@
 		</div>
 	</div>
 	<?php endif; ?>
+
+	<?php $early_adopters = get_field('early_adopters');
+		if (!empty($early_adopters )) : ?>
+		<div class="bg-theme-gray d-flex flex-column py-26">
+			<div class="w-326 w-md-641 w-xl-840 mx-auto">
+				<h3 class="text-theme-primary font-heebo-bold fz-28 fz-md-36 leading-40 mb-0 mb-6 mb-md-14 mb-xl-18"><?php echo $early_adopters['heading']; ?></h3>
+				<p class="mb-0 font-heebo-regular fz-18 leading-27"><?php echo $early_adopters['sub_heading']; ?></p>
+				<a class="glific-button-border px-9.5 bg-theme-secondary py-4 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none mt-6 d-block w-233" href="<?php echo $early_adopters['case_study_link']['link']; ?>"><?php echo $early_adopters['case_study_link']['button_text']; ?>
```

Comment:
@mohitgusain can we divide these long lines into multiple lines. It would be better to not use more than 80-100 characters in a single line.
```suggestion
				<a class="glific-button-border px-9.5 bg-theme-secondary py-4 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none mt-6 d-block w-233"
				    href="<?php echo $early_adopters['case_study_link']['link']; ?>" >
				    <?php echo $early_adopters['case_study_link']['button_text']; ?>
```